src/NServiceBus.Persistence.ServiceFabric/Outbox/OutboxStorage.cs

```diff
@@ -59,7 +59,7 @@ public Task<OutboxTransaction> BeginTransaction(ContextBag context)
                 operations[i] = new StoredTransportOperation(t.MessageId, t.Options, t.Body, t.Headers);
             }
 
-            var tx = ((ServiceFabricOutboxTransaction) transaction).Transaction.Value;
+            var tx = ((ServiceFabricOutboxTransaction) transaction).Transaction.Transaction;
```

Comment:
Not really proud of this train wreck. But since it was previously also a train wreck I figured I'll leave it at that. If you can come up with any good naming ideas feel free to shoot.